frontend/src/scenes/project/Settings/index.tsx

```diff
@@ -32,6 +32,7 @@ import { LemonTag } from 'lib/components/LemonTag/LemonTag'
 import { AuthorizedUrlsTable } from 'scenes/toolbar-launch/AuthorizedUrlsTable'
```

Comment:
Unrelated: guess how many [coding conventions](https://posthog.com/handbook/engineering/conventions/frontend-coding#do-s--dont-s) this file violates 😇